package.json

```diff
@@ -30,7 +30,9 @@
   },
   "dependencies": {
     "lodash": "^4.17.4",
+    "md5": "^2.2.1",
     "node-fetch": "^2.1.2",
+    "temp": "^0.9.1",
```

Comment:
This looks like it is here by accident?

Reply:
Yes, `npm uninstall` didn't remove that hehe. Thanks for pointing that out, would've added an extra mess to`node_modules` :D